sdk/lib/drivers/wdf/shared/inc/private/common/fxobject.hpp

```diff
@@ -319,7 +319,7 @@ class FxObject {
     union {
         //
         // This field is set when the object is being contructed or before
-        // Commit() and from then on is read only.
+        // Commit() and from then on is read-only.
```

Comment:
```suggestion
        // Commit() and from then on is read only.
```

Reply:
I think here 'read-only' is an adjective, so the PR commit is correct....?

Reply:
yes exactly, because 3rd-party :D